Old Bailey Online.js

```diff
@@ -296,6 +297,40 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.oldbaileyonline.org/browse.jsp?id=t16780828-12&div=t16780828-12&terms=hog#highlight",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": ".",
```

Comment:
That doesn't seem right/helpful as data import. I think what I would do is strip all trailing periods and commas, and then set `newItem.title = [no title]` when there's nothing left -- does that sound right?